src/components/Button.js

```diff
@@ -70,4 +72,22 @@ export const NavigationButton = styled(Button)`
   color: var(--background);
 `
 
+export const AddRecipeTitleToMealplanToggle = styled(Button)`
+  width: auto;
+  height: auto;
+  margin-right: 10px;
+  padding: 5px 5px 5px 5px;
```

Comment:
Wenn du in alle Richtungen 5px hast, kannst du es auch einfach nur einmal schreiben. `padding: 5px`

Reply:
Upsi. Jap. Das macht Sinn. Da hatte ich recht lange mit unterschiedlichen Werten rumprobiert und es am Ende doch gleichmäßig gemacht. 😅